src/applications/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -71,24 +70,36 @@ class DocumentRequestPage extends React.Component {
     this.props.getClaimDetail(this.props.claim.id);
     this.props.router.push(`your-claims/${this.props.claim.id}/files`);
   }
+
+  renderBreadcrumbs(claim, trackedItem) {
+    const crumbs = [
+      <a href="/" key="home">Home</a>,
+      <a href="/disability-benefits/" key="disability-benefits">Disability Benefits</a>,
+      <Link to="/" key="claims-home">Track Your Claims and Appeals</Link>,
+      <Link to={`/your-claims/${claim.id}`} key="claim-id">Status Details</Link>,
+      <Link to={`/your-claims/${claim.id}/document-request/${trackedItem}`} key="claim-id">Document Request</Link>
+    ];
+
+    return crumbs;
+  }
+
```

Comment:
This seems pretty different than the previous breadcrumbs. Were there new requirements here? Just confirming that it's no longer necessary to link to `/closed` anymore based on `this.props.claim.attributes.open`. It also looks like the title of `claims-home` changed from `Your Claims` to `Track Your Claims and Appeals`.

Reply:
@ncksllvn The name change was coordinated with the Content team. Removing `/closed` was an oversight, and based on the wider conversation to consolidate the app's breadcrumb using Formation, will be restored.